multiversx_sdk_cli/cli_contracts.py

```diff
@@ -52,6 +52,7 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
     sub.add_argument("--directory", default="scenarios",
                      help="🗀 the directory containing the tests (default: %(default)s)")
     sub.add_argument("--wildcard", required=False, help="wildcard to match only specific test files")
+    _add_recursive_arg(sub)
```

Comment:
In the CLI tests, you can add `--recursive` on one of them, so that we have the flow covered:

https://github.com/multiversx/mx-sdk-py-cli/blob/main/multiversx_sdk_cli/tests/test_cli_contracts.sh#L62